app.js

```diff
@@ -22,4 +22,24 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers().then((beerList) => {
+    console.log(beerList)
+    res.render('beers', {beers: beerList})
+  }).catch(error => {
+    console.log("error in beers: ", error)
+  })
+  })
+
+app.get('/random-beers', (req, res) => {
+  punkAPI.getRandom().then((responseFromAPI) => {
+    res.render('random-beers', {randomBeer: responseFromApi});
```

Comment:
here you are not sending the correct data that you are getting from the API.
`responseFromAPI` !== `responseFromApi`
remembering that javascript is case sensitive

another thing is the name of the view `random-beers` that is different from the view that you have `random-beer`